docs/lib/datastore/fragments/js/sync/50-selectiveSync.md

```diff
@@ -39,23 +46,26 @@ DataStore.configure({
     })
   ]
 });
-```
-And you want to change the filter that gets applied at runtime, you can do the following:
-```js
+
 async function changeSync() {
   rating = 1;
   await DataStore.stop();
   await DataStore.start();
 }
 ```
 
-Upon calling `start()` (or executing a DataStore operation, e.g., `query`, `save`, `delete`, or `observe`), DataStore will reevaluate the `syncExpressions`. 
+Upon calling `DataStore.start()` (or executing a DataStore operation, e.g., `query`, `save`, `delete`, or `observe`), DataStore will reevaluate the `syncExpressions`. 
 
 In the above case, the predicate will contain the value `1`, so all Posts with `rating > 1` will get synced down.
 
-Please note that when calling `DataStore.stop()` the existing contents of the local store are retained. 
+Keep in mind that the existing contents of the local store is _retained_ when calling `DataStore.stop()`.
+
+<amplify-callout>
+
+If you're applying a more restrictive filter and would therefore like to clear the local records, use `DataStore.clear()` instead:
```

Comment:
When applying a more restrictive filter, clear the local records first by running `DataStore.clear()`: